resources/zib/zib-BodyHeight.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-BodyHeight" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-BodyHeight" />
+  <name value="ZibBodyHeight" />
+  <title value="zib BodyHeight" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A person’s body height." />
+  <purpose value="This Observation resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) BodyHeight v3.1.1(2020)](https://zibs.nl/wiki/BodyHeight-v3.1.1(2020EN)).&#xD;&#xA;&#xD;&#xA;This profile tries to be compatible with, though is not derived from, the relevant [HL7 Vital Signs](https://www.hl7.org/fhir/observation-vitalsigns.html) profile." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-bodyheight-v3.1.1-2020EN" />
+    <uri value="https://zibs.nl/wiki/BodyHeight-v3.1.1(2020EN)" />
+    <name value="zib BodyHeight-v3.1.1(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
+      <path value="Observation" />
+      <short value="BodyHeight" />
+      <alias value="Lichaamslengte" />
+      <mapping>
+        <identity value="zib-bodyheight-v3.1.1-2020EN" />
+        <map value="NL-CM:12.2.1" />
+        <comment value="BodyHeight" />
+      </mapping>
+    </element>
+    <element id="Observation.code.coding.system">
+      <path value="Observation.code.coding.system" />
+      <fixedUri value="http://loinc.org" />
```

Comment:
* Kunnen we hiervoor geen pattern gebruiken zoals we op andere plekken ook voornamelijk gebruiken?
* In het STU3 profiel en in de vitalsigns profielen staat de code op een eigen slice...
* In het STU3 profiel heb je ook nog een code voor standing en lying... wat gebeurt daarmee?

Reply:
Using core extension:

If that's not possible, check if HL7 provides an extension to implements the concept.